importScripts/sampleRecords/itemData.destination.json

```diff
@@ -11,7 +11,7 @@
       "ca_objects.PhysicalCondition":"Fair \/ Used",
       "ca_objects.Materials":"Oil on pre-grounded Windsor &amp; Newton canvas.",
       "ca_objects.Dimensions.DimensionsValue":"H:44.9cm W:29.6cm",
-      "classificationHierarchy":"Root node for Classification (178); ILLUSTRATIONS (233); Oil (234); portrait (235)"
+      "classificationHierarchy":"Root node for Classification (179); ILLUSTRATIONS (224); Oil (225); portrait (226)"
```

Comment:
New source records are being imported for this test, which creates a new item id in the classification, hence these values are now renumbered.